src/tests/group-example/src/main.rs

```diff
@@ -1361,11 +1334,11 @@ async fn main_run() {
             object: NONObjectInfo::new(proposal.desc().object_id(), buf, Some(proposal_any)),
             access: Some(AccessString::full()),
         };
-        noc.put_object(req).await;
+        noc.put_object(req).await.expect("put proposal failed.");
```

Comment:
The error message should contain enough context, such as id, name, some value, etc.